### DISCUSSION TITLE
BorgBackup on Hetzner Storage

### DISCUSSION THREAD

**Discussion by iamshoXy** [WeblateOrg/weblate#11391] [Q&A] [answered]
Hey, I have a Hetzner Storage where I would like to store the backups on. I've added the SSH Key from Weblate on the storage server. However, I am running into this issue right now:
```
Remote: ssh: connect to host uXXXX.your-storagebox.de port 23: Network is unreachable
Connection closed by remote host. Is borg working on the server?
```

**Backup repository URL**
`ssh://uXXXX@uXXXX.your-storagebox.de:23/weblate-backup`

I can access the storage box from my computer and the server where weblate is running on.
Do I actually have to install BorgBackup on the storage box? I don't think Hetzner allows me to do so.

Any help is appreciated!

**Comment by nijel** [comment, marked as answer]
This looks more like a networking issue. borgbackup is installed on Hetzner on port 23.

**Comment by iamshoXy** [comment]
I don't know why it didn't work, but it does now.. 👌

**Reply by nijel** [reply]
Sometimes their DNS take some time to propagate, so maybe it was it.
